NuGet.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
We don't add these feeds directly, we mirror the packages to our feed. @ryanbrandenburg

Reply:
Yeah, `System.ServiceProcess.ServiceController` is already being mirrored to `aspnetcore-ci-dev`.